client/packages/system/src/Stock/Components/StockLineForm.tsx

```diff
@@ -70,6 +70,16 @@ export const StockLineForm: FC<StockLineFormProps> = ({ draft, onUpdate }) => {
     }
   };
 
+  useEffect(() => {
+    function handleKeyDown(this: HTMLElement, ev: KeyboardEvent) {
+      if (ev.ctrlKey && ev.key === 's') {
+        scanBarcode();
+      }
+    }
+    document.body.addEventListener('keydown', handleKeyDown);
+    return () => document.body.removeEventListener('keydown', handleKeyDown);
+  }, []);
+
```

Comment:
One thing I noticed, probably for a new issue. In electron, in the StockLine modal, when trying to launch the scanner when there's no hardward scanner present, it goes to a full error page:
![Screenshot 2023-05-15 at 9 38 45 AM](https://github.com/openmsupply/open-msupply/assets/5456533/c5de4613-87b6-49b2-95df-1fcc61ec8843)

Whereas in the browser, and in the Outbound shipment page (on both browser and electron) it gives the nice Snackbar notification as expected.

The code looks like it should be fine -- display the notification on error, so it's probably worth looking into at some point.

Reply:
The error page is fine in the compiled version - this is what I'm getting in electron:

<img width="885" alt="Screenshot 2023-05-15 at 12 13 48 PM" src="https://github.com/openmsupply/open-msupply/assets/9192912/ef60047f-52f3-457d-ad32-a4b062b2da39">